active_vision: true
closest_point_bound: 0.3
sigma_scale_upper_bound: 10.0
perturb_standard_deviation: [0.05, 0.05, 0.05]
fixation_point_error_tolerance: 0.003
gaze_timeout: 10
